plugins/kubernetes/src/KubernetesContent.tsx

```diff
@@ -64,7 +64,7 @@ export const KubernetesContent = ({
 
   return (
     <Page themeId="tool">
-      <Content>
+      <Content noPadding>
```

Comment:
Hold up, this component has no business being a `Page` and using `Content` in the first place does it?

Reply:
Ohh good point, I don't know 🧐

I think having a content inside the tabs make some sense but not sure about the page